tsconfig.json

```diff
@@ -36,6 +36,7 @@
       "utils": ["src/utils"],
       "services*": ["src/services/*"],
       "styles": ["src/styles"],
+      "comparision": ["io-fdk/comparision"]
```

Comment:
Path should be src/io-fdk
comparision key should be changed to io-fdk and also same for path